src/index.html

```diff
@@ -7,6 +7,48 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="week">
+      <div class="week__day">Monday</div>
+      <div class="week__day">Tuesday</div>
+      <div class="week__day">Wednesday</div>
+      <div class="week__day">Thursday</div>
+      <div class="week__day">Friday</div>
+      <div class="week__day">Saturday</div>
+      <div class="week__day">Sunday</div>
+    </div>
+
+    <div class="calendar calendar--month-length_31">
+      <div class="calendar__day calendar__day--start-day-sun"></div>
```

Comment:
please, improve modifier names)

[FUNCTIONALITY] - Your project should work correctly (have 31 days and start on Monday) with invalid modifier values, **like start-day--ghy or days--27.**

```suggestion
    <div class="calendar calendar--month-length--31">
      <div class="calendar__day calendar__day--start-day--sun"></div>
```